contrib/imgprune/entrypoint.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -e
+
+source /usr/local/bin/lib_entrypoint.sh
+
+req_env_var GCPJSON GCPNAME GCPPROJECT IMGNAMES
+
+gcloud_init
+
+# Different purpose from quay.io/libpod/imgts
+EXCLUDE="$IMGNAMES $IMAGE_BUILDER_CACHE_IMAGE_NAME"
+# For safety's sake + limit nr background processes
+PRUNE_LIMIT=10
+THEFUTURE=$(date --date='+1 hour' +%s)
+THRESHOLD=$(date --date='90 days ago' +%s)
+FORMAT='value[quote](name,selfLink,creationTimestamp,labels)'
+PROJRE="/v1/projects/$GCPPROJECT/global/"
+BASE_IMAGE_RE='cloud-base'
+FILTER="selfLink~$PROJRE AND creationTimestamp<$(date --date='30 days ago' --iso-8601=date) AND NOT name=($EXCLUDE)"
+TODELETE="/tmp/todelete"
```

Comment:
Eeeeek; please use `mktemp --tmpdir todelete.XXXXXX`. (I know it almost certainly doesn't matter in this case, but proper habits are important)

Reply:
got it